core/src/main/java/org/web3j/protocol/core/JsonRpc2_0Web3j.java

```diff
@@ -543,7 +543,7 @@ public Request<?, EthLog> ethGetFilterChanges(BigInteger filterId) {
     public Request<?, EthLog> ethGetFilterLogs(BigInteger filterId) {
         return new Request<>(
                 "eth_getFilterLogs",
-                Arrays.asList(Numeric.toHexStringWithPrefixSafe(filterId)),
+                Arrays.asList(Numeric.toHexStringWithPrefix(filterId)),
                 web3jService,
```

Comment:
Three methods are still using toHexStringWithPrefix
shhUninstallFilter, shhGetMessages, shhGetFilterChanges
Let's do it for all of them.